samples/zoom-bar.html

```diff
@@ -76,10 +80,12 @@
                     },
                     pan: {
                         enabled: true,
-                        mode: 'y'
+                        mode: 'x',
+                        speed: 10,
+                        threshold: 10
                     },
                     zoom: {
-                        enabled: true,
+                        enabled: false,
```

Comment:
This file is `zoom-bar.html`. I think it would be best to create a new pan only bar sample `pan-bar.html`